src/ResourceManager/Sql/Commands.Sql/Security/Cmdlet/SqlCmdletBase.cs

```diff
@@ -37,47 +41,57 @@ public abstract class SqlDatabaseSecurityCmdletBase : SqlDatabaseCmdletBase
         [Parameter(Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = "The name of the resource group")]
         [ValidateNotNullOrEmpty]
         public string ResourceGroupName { get; set; }
-
+        
         /// <summary>
+        /// Gets or sets the name of the database server to use.
+        /// </summary>
+        [Parameter(Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = "SQL Database server name.")]
```

Comment:
All of the cmdlets that would target a specific sql resource would require the server name. The cmdlets that look at the sql resources from the outside(which I assume would be very few of them) can override this parameter and set it as optional. I think that defaulting this property to be required would cover the vast majority of the sql cmdlets.